bump.js

```diff
@@ -46,8 +47,13 @@ module.exports = async function ({ context, inputs }) {
   ])
   const branchName = `release/${newVersion}`
 
+  const messageTemplate = inputs['commit-message'] || 'Release {version}'
```

Comment:
the default will come from the action definition, we don't have to be redundant here

Reply:
That was there because the commit-message input is not required so the user could just leave a white space. Maybe that's okay and we can remove this check? 

Or we could set the commit-message input as required and the user can just use the default value if they don't want to modify it like the semver input.

https://github.com/nearform/optic-release-automation-action/blob/a60f257963910ea7c3d7a389accbe7abbb506a87/README.md#L35-L38

What do you think @simoneb ?

Reply:
well then we have an inconsistency between readme and the action definition https://github.com/nearform/optic-release-automation-action/pull/34/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R45

Reply:
I have updated the documentation and behaviour to make the input not required and fall back to the default value in the action definition